crates/ra_syntax/src/ast/traits.rs

```diff
@@ -69,16 +75,42 @@ pub trait ModuleItemOwner: AstNode {
 #[derive(Debug)]
 pub struct ItemOrMacroIter(SyntaxNodeChildren);
 
+fn collect_derive_attr(item: ast::ModuleItem) -> Option<ItemOrMacro> {
```

Comment:
this is called `collect_derive_attr`, but doesn't seem to do anything specific to derives?

Reply:
Also, I'm wondering whether this means that we'll treat any item with an attribute as a macro call...

Reply:
Yes, we're currently treating every attribute on def items (structs and enums) as a macro call.